src/routes/resource-edit.js

```diff
@@ -35,47 +46,89 @@ class ResourceEditRoute extends Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    let { packageName, packageId } = prevProps.model;
+    if (!prevProps.model.destroy.isResolved && this.props.model.destroy.isResolved) {
+      this.context.router.history.replace(`/eholdings/packages/${packageId}?searchType=packages&q=${packageName}`,
+        { eholdings: true, isDestroyed: true });
+    }
+  }
+
   resourceEditSubmitted = (values) => {
-    let { model, updateResource } = this.props;
+    let { model, updateResource, destroyResource } = this.props;
     let {
       coverageStatement,
       customCoverages,
       customEmbargoValue,
       customEmbargoUnit,
       customUrl
     } = values;
-    model.customCoverages = customCoverages.map((dateRange) => {
-      let beginCoverage = !dateRange.beginCoverage ? null : moment(dateRange.beginCoverage).tz('UTC').format('YYYY-MM-DD');
-      let endCoverage = !dateRange.endCoverage ? null : moment(dateRange.endCoverage).tz('UTC').format('YYYY-MM-DD');
 
-      return {
-        beginCoverage,
-        endCoverage
+    if (values.isSelected === false && model.package.isCustom) {
+      destroyResource(model);
+    } else if (values.isSelected === false) {
+      model.isSelected = !model.isSelected;
+      model.customCoverages = [];
+      model.contributorsList = [];
+      model.visibilityData.isHidden = false;
+      model.identifiersList = [];
+      model.identifiers = {};
```

Comment:
Curious as to why there's both an `identifiersList` and and `identifiers`. Also: `contributors` and `identifiers` are both title, not resource, attributes. Do we need to do anything with those here?

Reply:
It's interesting because it was throwing errors when trying to update about all of these. I can double check but the error seemed to vary depending on which title you were trying to deselect.

Reply:
A resource is comprise of a Title. So what comes through the resource edit route could contain title level attrs from both a custom or managed-title?

Reply:
It throws a error that identifierList is not empty but when you check the Error is has a pointer to identifiers.....
<img width="755" alt="screen shot 2018-05-04 at 5 44 40 pm" src="https://user-images.githubusercontent.com/1953098/39654494-0bd06058-4fc3-11e8-88d5-e2911872a08e.png">